.github/autobuild/linux_deb.sh

```diff
@@ -29,6 +29,13 @@ setup() {
     sudo apt-get -qq update
     sudo apt-get -qq --no-install-recommends -y install devscripts build-essential debhelper fakeroot libjack-jackd2-dev qtbase5-dev qttools5-dev-tools qtmultimedia5-dev
 
+    # The debian build scripts will update config.{guess,sub} and will therefore cause the Jamulus.pro
```

Comment:
Why does it do this?

And if they do need to do this _before_ building the binary, tough luck - it's dirty.

Reply:
> Why does it do this?

It's an optimization. Distributions commonly run autoreconf themselves or substitute those files with pre-generated content. I've found a different, better approach though and have updated this PR. I found a way to disable this debian-specific logic and this looks way cleaner.